### PR TITLE
Documentation Refactor 

### PR DESCRIPTION
**Pull request by kris-nova**
# Major documentation change and cleanup for kops
-  Adding new tags
-  Pulling notes out of README
-  Adding gifcast (yey!)
-  Trying to make the repo look new and shiny 

### Check out the new repo proposal [here](https://github.com/DualSpark/kops/tree/readme-cleanups)

### CODE REVIEW

**Comment by kris-nova** [comment]
This will close https://github.com/kubernetes/kops/issues/641

**Comment by kris-nova** [comment]
More work for https://github.com/kubernetes/kops/pull/721

File: README.md [hunk 1]
```diff
@@ -1,151 +1,72 @@
...
-  so that domains will resolve.  You can reuse an existing domain name (e.g. `mydomain.com`), or you can create
-  a "child" hosted zone (e.g. `myclusters.mydomain.com`) if you want to isolate them.  Note that with AWS Route53,
-  you can have subdomains in a single hosted zone, so you can have `cluster1.testclusters.mydomain.com` under
-  `mydomain.com`.
+We like to think of it as `kubectl` for clusters. 
 
-* Pick a DNS name under this zone to be the name of your cluster.  kops will set up DNS so your cluster
-  can be reached on this name.  For example, if your zone was `mydomain.com`, a good name would be
-  `kubernetes.mydomain.com`, or `dev.k8s.mydomain.com`, or even `dev.k8s.myproject.mydomain.com`. We'll call this `NAME`.
+The tool enables the user to deploy functional Kubernetes clusters in the cloud ([AWS](/docs/aws.md) or GCE).
```
Comment (justinsb) [inline README.md]: We might want to wait before promising GCE... it's still turned off.

Comment (justinsb) [inline README.md]: How about "kubectl lets you deploy..."

File: README.md [hunk 2]
```diff
@@ -1,151 +1,72 @@
...
-export KOPS_STATE_STORE=s3://<somes3bucket>
-# NAME=<kubernetes.mydomain.com>
-${GOPATH}/bin/kops export kubecfg ${NAME}
-```
-
-You can now use kubernetes using the kubectl tool (after allowing a few minutes for the cluster to come up):
-
-```kubectl get nodes```
+* HA (Highly Available) Kubernetes clusters
+* Automated Kubernetes cluster [CRUD](/docs/commands.md) for the cloud ([AWS](/docs/aws.md) and GCE)
```
Comment (justinsb) [inline README.md]: Let's pull GCE here also - for now :-)

File: README.md [hunk 3]
```diff
@@ -1,151 +1,72 @@
...
+* Uses a state-sync model for **dry-run** and automatic **idempotency**
+* Custom support for `kubectl` [add-ons](docs/addons.md)
+* Can produce configurations in other formats 
+    * [Terraform configuration](/docs/terraform.md)
+    * Cloud-Init configuration
```
Comment (justinsb) [inline README.md]: FYI, I'm thinking about removing cloud-init support also as it doesn't seem anyone cares to use it.

File: docs/commands.md [hunk 4]
```diff
@@ -29,3 +29,22 @@ when you are happy that it is deleting the right things you run `kops delete clu
 ## `kops version`
 
 `kops version` will print the version of the code you are running.
+
+## Other interesting modes:
```
Comment (justinsb) [inline docs/commands.md]: Can we make this section more discoverable?  Maybe in the aws.md guide (even if we then have to duplicate it for gce...)

File: README.md [hunk 5]
```diff
@@ -1,151 +1,72 @@
...
-  `mydomain.com`.
+We like to think of it as `kubectl` for clusters. 
 
-* Pick a DNS name under this zone to be the name of your cluster.  kops will set up DNS so your cluster
-  can be reached on this name.  For example, if your zone was `mydomain.com`, a good name would be
-  `kubernetes.mydomain.com`, or `dev.k8s.mydomain.com`, or even `dev.k8s.myproject.mydomain.com`. We'll call this `NAME`.
+The tool enables the user to deploy functional Kubernetes clusters in the cloud ([AWS](/docs/aws.md) or GCE). 
 
-* Set `AWS_PROFILE` (if you need to select a profile for the AWS CLI to work)
+We encourage everyone to read more about [how it works](/docs/how_it_works.md) or check out some of the [other interesting modes](/docs/commands.md#other-interesting-modes).
```
Comment (justinsb) [inline README.md]: Can we a link to aws.md more prominent.  It is why people are here.

Maybe: "Quickstart: Launching a kubernetes cluster on AWS"

File: README.md (hunk 1 above)
Comment (kris-nova) [inline README.md]: Assuming you mean kops not kubectl?

File: docs/commands.md (hunk 4 above)
Comment (kris-nova) [inline docs/commands.md]: Added another pointer in the *Other Resources* section 
Added a pointer in *aws.md*

**Comment by kris-nova** [comment]
@justinsb - @chrislovecnm 
 
Any more comments?

File: README.md [hunk 6]
```diff
@@ -1,151 +1,75 @@
...
+We like to think of it as `kubectl` for clusters. 
 
-Build the code (make sure you have set GOPATH):
-```
-go get -d k8s.io/kops
-cd ${GOPATH}/src/k8s.io/kops/
-git checkout release
-make
-```
+kops lets you deploy functional Kubernetes clusters in the cloud. Right now kops focuses on deploy prod
```
Comment (justinsb) [inline README.md]: Did we lose a line here?

**Comment by kris-nova** [comment]
@justinsb - Line typo fixed, LMK if we want a rebase

**Comment by chrislovecnm** [comment]
I have two pending comments at the top

**Comment by kris-nova** [comment]
@chrislovecnm - am I missing something? I am not showing any open comments from you :/

File: README.md [hunk 7]
```diff
@@ -1,151 +1,72 @@
...
+* Easy command line syntax
+* Community support
 
-## Cluster management
+### Example on AWS
 
-* Set up [add-ons](docs/addons.md), to add important functionality to k8s.
+<p align="center">
+  <img src="/docs/img/demo.gif"> </image>
+</p>
```
Comment (kris-nova) [inline README.md]: That is relative - markdown `HTML`s differently than markdown `markdown`s

**Comment by kris-nova** [comment]
@chrislovecnm - your comments have been addressed - thanks for the input

**Pull request merged**